### PR TITLE
Change cluster-api-provider-baremetal repo path

### PR DESCRIPTION
**Pull request by vrutkovs**
Paths need to be updated after this repo was moved to openshift org.

### CODE REVIEW

**Comment by vrutkovs** [comment]
Current state:
```
$ make test            
go generate ./pkg/... ./cmd/...
go fmt ./pkg/... ./cmd/...
go vet ./pkg/... ./cmd/...
go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go all
CRD manifests generated under '/home/vrutkovs/go/src/github.com/openshift/cluster-api-provider-baremetal/config/crds' 
RBAC manifests generated under '/home/vrutkovs/go/src/github.com/openshift/cluster-api-provider-baremetal/config/rbac' 
kustomize build config/default/ > provider-components.yaml
Error: rawResources failed to read Resources: Load from path ../rbac/rbac_role.yaml failed: security; file '../rbac/rbac_role.yaml' is not in or below '/var/home/vrutkovs/go/src/github.com/openshift/cluster-api-provider-baremetal/config/default'

make: *** [Makefile:36: manifests] Error 1
```

**Comment by russellb** [comment]
@vrutkovs tl;dr - you need kustomize < 2.0

https://github.com/metalkube/cluster-api-provider-baremetal/issues/5

File: config/kustomization.yaml [hunk 1]
```diff
@@ -18,22 +18,22 @@ namePrefix: cluster-api-provider-baremetal-
 # YAML string, with resources separated by document
 # markers ("---").
 resources:
-- ../rbac/rbac_role.yaml
-- ../rbac/rbac_role_binding.yaml
-- ../manager/manager.yaml
+- rbac/rbac_role.yaml
+- rbac/rbac_role_binding.yaml
+- manager/manager.yaml
```
Comment (russellb) [inline config/kustomization.yaml]: Can we drop this to minimize changes with the upstream repo?

Comment (russellb) [inline config/kustomization.yaml]: Can we drop this to minimize changes with the upstream repo?

Comment (vrutkovs) [reply config/kustomization.yaml]: Fixed

**Comment by vrutkovs** [comment]
Current state - works locally, but breaks on CI:
```
2019/03/11 12:36:14 Executing test unit
go generate ./pkg/... ./cmd/...
go fmt ./pkg/... ./cmd/...
go vet ./pkg/... ./cmd/...
pkg/apis/addtoscheme_baremetal_v1alpha1.go:20:2: cannot find package "github.com/metalkube/cluster-api-provider-baremetal/pkg/apis/baremetal/v1alpha1" in any of:
	/go/src/sigs.k8s.io/cluster-api-provider-baremetal/vendor/github.com/metalkube/cluster-api-provider-baremetal/pkg/apis/baremetal/v1alpha1 (vendor tree)
	/usr/local/go/src/github.com/metalkube/cluster-api-provider-baremetal/pkg/apis/baremetal/v1alpha1 (from $GOROOT)
	/go/src/github.com/metalkube/cluster-api-provider-baremetal/pkg/apis/baremetal/v1alpha1 (from $GOPATH)
pkg/controller/add_machine.go:20:2: cannot find package "github.com/metalkube/cluster-api-provider-baremetal/pkg/cloud/baremetal/actuators/machine" in any of:
	/go/src/sigs.k8s.io/cluster-api-provider-baremetal/vendor/github.com/metalkube/cluster-api-provider-baremetal/pkg/cloud/baremetal/actuators/machine (vendor tree)
	/usr/local/go/src/github.com/metalkube/cluster-api-provider-baremetal/pkg/cloud/baremetal/actuators/machine (from $GOROOT)
	/go/src/github.com/metalkube/cluster-api-provider-baremetal/pkg/cloud/baremetal/actuators/machine (from $GOPATH)
cmd/manager/main.go:25:2: cannot find package "github.com/metalkube/cluster-api-provider-baremetal/pkg/apis" in any of:
	/go/src/sigs.k8s.io/cluster-api-provider-baremetal/vendor/github.com/metalkube/cluster-api-provider-baremetal/pkg/apis (vendor tree)
	/usr/local/go/src/github.com/metalkube/cluster-api-provider-baremetal/pkg/apis (from $GOROOT)
	/go/src/github.com/metalkube/cluster-api-provider-baremetal/pkg/apis (from $GOPATH)
make: *** [vet] Error 1
```
https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_release/3055/rehearse-3055-pull-ci-openshift-cluster-api-provider-baremetal-master-unit/2/build-log.txt

**Comment by vrutkovs** [comment]
Fixed by CI config

**Pull request closed without merging**